pyproject.toml

```
@@ -18,6 +18,16 @@ select = [
     "B"
 ]
 
+[tool.codespell]
+# note: pre-commit passes explicit lists of files here, which this skip file list doesn't override -
+# this is only to allow you to run codespell interactively
+skip = "./.git,./build,./python/_skbuild/,.*egg-info.*,./.mypy_cache,./pyproject.toml,./benchmarks/utilities/cxxopts.hpp"
```

```suggestion
skip = "./.git,./build,./.mypy_cache,./pyproject.toml"
```

This project doesn't have some of these paths... I'm guessing that maybe you just copied this from another repo. Let's please keep it limited to directory that appear in this project.